Ch heroku hosting 163505957

kenzdozz opened the pull request:
#### What does this PR do?
Hosted the app on heroku
#### Description of Task to be completed?
Adjusted the server to use environment port if available.
Pushed app to heroku
#### How should this be manually tested?
API endpoints can be tested on: https://poli-tico.herokuapp.com/
#### What are the relevant pivotal tracker stories?
163505957

ascii-dev reviewed (approved):
I think this LGTM, but you still need to fix the multiple duplicate commits.

kenzdozz committed:
chore(heroku-hosting): Hosting the App on Heroku
- Hosting the App on Heroku and making it accessible through a URL
[Delivers #163505957]

kenzdozz committed:
Merge branch 'develop' into ch-heroku-hosting-163505957

ascii-dev reviewed (approved):
LGTM
